src/metaswitch/clearwater/queue_manager/main.py

```diff
@@ -37,24 +37,27 @@
 Usage:
   main.py --local-ip=IP --local-site=SITE --etcd-key=KEY --node-type=TYPE
           [--foreground] [--log-level=LVL] [--log-directory=DIR] [--pidfile=FILE]
+          [--ignore-plugin-responses=RESP]
 
 Options:
-  -h --help                   Show this screen.
-  --local-ip=IP               IP address
-  --local-site=NAME           Local site name
-  --etcd-key=KEY              Etcd key (top level)
-  --node-type=TYPE            Node type (e.g. Sprout, AIO, ...)
-  --foreground                Don't daemonise
-  --log-level=LVL             Level to log at, 0-4 [default: 3]
-  --log-directory=DIR         Directory to log to [default: ./]
-  --pidfile=FILE              Pidfile to write [default: ./config-manager.pid]
+  -h --help                      Show this screen.
+  --local-ip=IP                  IP address
+  --local-site=NAME              Local site name
+  --etcd-key=KEY                 Etcd key (top level)
+  --node-type=TYPE               Node type (e.g. Sprout, AIO, ...)
+  --foreground                   Don't daemonise
+  --log-level=LVL                Level to log at, 0-4 [default: 3]
+  --log-directory=DIR            Directory to log to [default: ./]
+  --pidfile=FILE                 Pidfile to write [default: ./config-manager.pid]
+  --ignore-plugin-responses=RESP Don't wait for plugin responses
```

Comment:
Is "ignore-plugin-responses" right?  Do we actually ignore plugin responses, or is it that we don't actually check service health?  Would check-health be a better name?  (Obviously, with the opposite sense!)